ComposeAdvanced/app/src/main/java/com/example/android/wearable/composeadvanced/presentation/ui/userinput/UserInputComponentsScreen.kt

```diff
@@ -38,41 +32,78 @@ import com.example.android.wearable.composeadvanced.R
 fun UserInputComponentsScreen(
     value: Int,
     onClickStepper: () -> Unit,
-    onClickSlider: () -> Unit
+    onClickSlider: () -> Unit,
+    onClickDemoDatePicker: () -> Unit,
+    onClickDemo12hTimePicker: () -> Unit,
+    onClickDemo24hTimePicker: () -> Unit,
 ) {
-    Column(
+    ScalingLazyColumn(
```

Comment:
I don't think we need this

Reply:
When I changed to Chip from CompactChip, we do.